examples/mobile/UIComponents/components/view-controls/main-tab/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="tabs">
+		<header>
+			<h1>
+				Main Tab
+			</h1>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview ui-content-area">
+				<li class="ui-li-anchor">
+					<a href="./main-tab-2-items.html">
+						2 elements
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="./main-tab-3-items.html">
+						3 elements
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="./main-tab-4-items.html">
+						4 elements
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="./main-tab-5-items.html">
+						5 elements
+					</a>
+				</li>
+			</ul>
+		</div>
+	</div>
+</body>
+
+</html>
```

Comment:
Please add a blank line at the end of the document.

Reply:
fixed